packages/elements/src/clock/index.ts

```diff
@@ -62,6 +68,8 @@ export class Clock extends ResponsiveElement {
     return VERSION;
   }
 
+  protected readonly defaultRole: string | null = 'group';
```

Comment:
There is no defaultRole for the component as the role is based on `interactive` attribute. Keep is as default (`null`)

Reply:
Updated by no role by default